application/pom.xml

```diff
@@ -86,6 +108,8 @@
                     <cmd>
                       <exec>
                         <arg>java</arg>
+                        <arg>-Dvertx.disableFileCaching=true</arg>
+                        <arg>-Dvertx.disableFileCPResolving=true</arg>
```

Comment:
Not a big expert on that but file caching and CPResolving seems to be related to cache resources file for fat jar and used a lot by the Vert.x web component. The problem with OpenShift is that they try to create a local folder for caching and it's something that can't be done on OpenShift because the container runs with a user without more privileges. An application running on OpenShift can write on an external volume (i.e. I set up one for the device registry file). Btw Clement from the Vert.x team suggested me to set vertx.CacheDirBase=/tmp where /tmp is another folder available for writing (without the need of a volume).